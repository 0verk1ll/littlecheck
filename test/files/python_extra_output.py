# RUN: /usr/bin/python %s

import sys

print("First")
# CHECK: First

print("Second")
