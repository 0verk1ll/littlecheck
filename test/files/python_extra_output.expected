Failure in python_extra_output.py:
Extra output after checks:
  Output line stdout:2: Second
